src/tools/gophish_import.py

```diff
@@ -246,7 +248,7 @@ def build_campaigns(api, assessment):
             logging.error(e)
             raise
 
-        logging.info(f"Campaign {campaign['name']} successfully loaded.\n")
+        logging.info("Campaign %s successfully loaded.\n", campaign["name"])
```

Comment:
This line still has an unnecessary `\n`.

Reply:
Its been removed. See 44e24edb9ac96eb42759bdea8a9229563811ca59